src/shared/database/setup.ts

```diff
@@ -190,4 +190,21 @@ async function setupNetworksTable(): Promise<void> {
         .catch((err: Error) => log.error(err.message))
     })
   }
+
+  const networksCount = await query('networks').countDistinct('id')
```

Comment:
@mvadari This one is to manually add a new network into the table if not already existed.

Reply:
That automatically happens by adding it to `networks`

Reply:
https://github.com/ripple/validator-history-service/blob/main/src/shared/database/setup.ts#L176-L186

Reply:
Notice that in #37 a new network was added without this being necessary

Reply:
There's an `if (!hasNetworks) {` condition there, so if the network table already existed, it will skip that part I believe

Reply:
Hm, that's fair - in that case, we shouldn't be doing this twice when the `networks` table is created, those two sections of the code should be combined.

Reply:
Combined them and tested locally.